src/components/Slot/Slot.tsx

```diff
@@ -57,6 +57,7 @@ class Slot extends UIComponent<Extendable<SlotProps>, any> {
 
   static create = createSlotFactory(Slot.defaultProps.as, content => ({ content }))
   static createHTMLInput = createSlotFactory('input', type => ({ type }))
+  static createHTMLListItem = createSlotFactory('li', children => ({ children }))
```

Comment:
no need to add this as we're removing HTML factories in #376
use `createSlotFactory('li', children => ({ children }))` directly where you need it